packages/koa-shopify-auth/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## [Unreleased] -->
 
+## [3.1.57] - 2020-05-01
```

Comment:
This needs to be a `3.1.61` since the last released version was [3.1.60](https://github.com/Shopify/quilt/blob/master/packages/koa-shopify-auth/package.json#L3). The changelogs to keep a good track of peer dep bumps via lerna 😢